docs/api/I13nUtils.md

```diff
@@ -0,0 +1,40 @@
+## I13nUtils
+
+The `I13Utils` mixin provides util functions for you to access the i13n nodes and execute i13n functionalities
```

Comment:
if we combine that all the functionalities in a mixin, that mixin will always created a i13nNode for that component, and adding everything seems heavy if we just want some functionalities like `executeI13nEvent` or `getI13nNode`, 

actually I didn't see any reason to expose I13nMixin to users, how about remove it from the document and get users to use createI13nNode anyway?